lib/rest/accounts/v1/credential/aws.ts

```diff
@@ -570,11 +529,13 @@ export function AwsListInstance(version: V1): AwsListInstance {
   };
 
   instance.page = function page(
-    params?: any,
-    callback?: any
+    params?:
+      | AwsListInstancePageOptions
+      | ((error: Error | null, item?: AwsPage) => any),
+    callback?: (error: Error | null, item?: AwsPage) => any
   ): Promise<AwsPage> {
```

Comment:
See a TS error here when strict mode is enabled

Reply:
Fixed! Typo with `item?` (should be `items`)